service/src/test/java/uk/gov/hmcts/reform/fpl/service/document/DocumentsListRendererTest.java

```diff
@@ -47,7 +66,7 @@ void shouldRenderDocumentBundlesMinimalSingleElement() {
                 .name("SWET")
                 .documents(List.of(DocumentView.builder()
                     .document(DocumentReference.builder()
-                        .filename("swet-doc.docx").url("fake-url.com").binaryUrl("test.com").build())
+                        .filename("swet-doc.docx").url("fake-url.com").binaryUrl("test.com/documents/docId").build())
```

Comment:
can you confirm whether it is `documents` or `documentsv2`?

Reply:
documents, and the renderer will convert it to documentsv2